src/main/java/uk/gov/hmcts/reform/bulkscan/services/BulkScanFL401AService.java

```diff
@@ -29,17 +45,49 @@ public BulkScanValidationResponse validate(
         BulkScanValidationResponse response =
                 bulkScanValidationHelper.validateMandatoryAndOptionalFields(
                         bulkScanValidationRequest.getOcrdatafields(),
-                        configManager.getValidationConfig(FormType.FL401A));
```

Comment:
update java doc at method level for both validate and transform

Reply:
Javadoc added.